examples/Cassie/cassie_rbt_state_estimator.h

```diff
@@ -10,24 +10,111 @@
 #include "systems/framework/output_vector.h"
 #include "systems/framework/timestamped_vector.h"
 #include "examples/Cassie/datatypes/cassie_out_t.h"
+#include "examples/Cassie/cassie_utils.h"
 
 namespace dairlib {
 namespace systems {
 
+using Eigen::Vector3d;
```

Comment:
Don't use `using` in a header. Everyone who `#incude`'s this header, would then be stuck with your `using` definitions.

Reply:
Done!